src/org/zaproxy/zap/model/Tech.java

```diff
@@ -160,4 +160,4 @@ public int compareTo(Tech o) {
         
         return this.toString().compareTo(o.toString());
     }
-}
+}
```

Comment:
Could you revert this or exclude this file?

Reply:
Of course, but it's strange because it does not change anything between the red and green lines.

Reply:
It removed a newline.